### ISSUE TITLE
Game should be able to provide a css style sheet

### ISSUE THREAD

**Issue by robartsd** [ArnoldSmith86/virtualtabletop#154]
While we do have the ability to add inline styles, I'd like to be able to specify a styles heet for the game. Game style sheet should load after the default styles but before any url style sheet. Game style sheet should be set per variant when editing games on the game list and stored in the variant's "_meta" object.

**Comment by robartsd** [comment]
We decided in meeting today that we would like to support this as an advanced feature.

**Comment by RaphaelAlvez** [comment]
This issue was added to Pick and Code. Note that the way to upload the css sheet should be flexible as we are transitioning to a new UI. The functionality shouldn't be affected by that though.

**Comment by robartsd** [comment]
I finally looked at the code used to serve up assets. The current strategy of checking the first character of the file to determine what the Content-type header should be will not work for CSS stylesheets because they don't have a particular character at the beginning.  As we've learned with the URL parameters, a CSS stylesheet is only applied if it is served with the correct Content-type.

Possible solutions:
- use a different scheme for stylesheet URLs than for image asset URLs
- add a marker to the beginning of uploaded stylesheets and strip it before serving up the file
